cbmc/proofs/DeserializeConnack/DeserializeConnack_harness.c

```diff
@@ -0,0 +1,34 @@
+/* Build using a config header, if provided. */
+#ifdef AWS_IOT_CONFIG_FILE
+    #include AWS_IOT_CONFIG_FILE
+#endif
+
+/* Standard includes. */
+#include <string.h>
+
+/* MQTT internal includes. */
+#include "private/aws_iot_mqtt_internal.h"
+
+AwsIotMqttError_t
+AwsIotMqttInternal_DeserializeConnack( const uint8_t * const pConnackStart,
+				       size_t dataLength,
+				       size_t * const pBytesProcessed );
```

Comment:
As discussed in the workshop preparation, I'd have a preference towards making this a comment only. Mainly because it suggests that this declaration is necessary, which is not the case.

Reply:
This is gone in the squashed commit.